src/Microsoft.Azure.WebJobs.Host/Hosting/IWebJobsStartup2.cs

```diff
@@ -0,0 +1,21 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+namespace Microsoft.Azure.WebJobs.Hosting
+{
+    /// <summary>
+    /// Interface defining a startup configuration action that should be performed
+    /// as part of host startup.
+    /// </summary>
+    public interface IWebJobsStartup2 : IWebJobsStartup
```

Comment:
Is there a reason you implement `IWebJobsStartup` in this interface instead of them being independent. Should the startup throw in the `Configure(IWebJobsBuilder builder)` if it requires the config?